netcore/CoreFX.issues.rsp

```diff
@@ -68,9 +60,6 @@
 ##  System.Diagnostics.TraceSource.Tests
 ####################################################################
 
-# requires precise gc (should be ignored in corefx)
```

Comment:
These changes appear to be unrelated.

Reply:
@vargaz I grouped them (and added a few more) in order to remove them all once we get https://github.com/dotnet/corefx/pull/39176 (turns out I should mark more tests with that attribute)